server/utils/auth.py

```diff
@@ -0,0 +1,117 @@
+from os import abort
```

Comment:
Feels like 'utils' is not quite the right place. We need some controllers dir

Reply:
Yes